samples/multiplatform/build.gradle.kts

```diff
@@ -0,0 +1,67 @@
+import org.gradle.kotlin.dsl.kotlin
+import org.jetbrains.kotlin.gradle.tasks.KotlinCompile
+
+plugins {
+    base
+}
+
+buildscript {
+    repositories {
+        mavenCentral()
+    }
+
+    dependencies {
+        classpath(kotlin("gradle-plugin", "1.2.31"))
+        classpath(kotlin("native-gradle-plugin", "0.6.2"))
```

Comment:
Use the `plugins{}` block for these dependencies

Reply:
I have no idea how to move these into plugins block, it just breaks the build for me